Makefile

```diff
@@ -59,9 +62,15 @@ check: bin/golangci-lint ## Check code quality (lint)
 	./bin/golangci-lint run --timeout 300s
 	cd test/_e2e && ../../bin/golangci-lint run --timeout 300s
 
+checkstyle: tools/goimports ## Check code style (imports and formatting)
+	@goimports -d $(CODECHECK)
+
 bin/golangci-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./bin v1.40.1
 
+tools/goimports:
+	@go get golang.org/x/tools/cmd/goimports
```

Comment:
Does this `go get` command create `tools/goimports`?  If not, I am unclear how this works as a Make target.  The idea is that the make target (in this case `tools/goimports`) executes if `tools/goimports` does not exist.  The idea being that it only fires once per call to `make checkstyle`